### DISCUSSION TITLE
mstump: description of idx in private functions

### DISCUSSION THREAD

**Discussion by NimaSarajpoor** [TDAmeritrade/stumpy#867] [General]
**In module mstump.py:**

The function `_mstump` calls the function `_compute_multi_D`  which has the argument `idx` with the following description:

```
idx : int
        The row index in `T`
```

The function `_mstump` also calls the function `core._compute_multi_PI` that has the same argument `idx` with the same description.

However, I think this description might be incorrect. If `T` is multi-dimensional, i.e. 2D numpy array, where each row corresponds to one time series, then the description of `idx` should be: 
```
the column index of `T`
```

This is based on what I can see, regarding `idx`, in the following lines of code:

https://github.com/TDAmeritrade/stumpy/blob/76cb9806dec2d32f80ff20a5a3c484c34b7a54bf/stumpy/mstump.py#L800-L814

**Comment by seanlaw** [comment]
I think you're right. Though, I think it's more like:

```
idx : int
    The subsequence index for the ith time series, `T[i]`
```

Is this better?

**Reply by NimaSarajpoor** [reply]
Yeah... I think it is clearer as it reflects a **subsequence** (that starts at `idx`)
